scripts/src/internal/services.ts

```diff
@@ -101,7 +102,12 @@ async function getPaymentJWT(order: db.Order, appId: string, user: User): Promis
 }
 
 export async function paymentComplete(payment: CompletedPayment) {
-	const order = await db.Order.getOne({ orderId: payment.id });
+	// if I can find the "paytment.id" in the redis, it means its a memo and the value is the real order id
+	const redis = getRedisClient();
+	const incomingOrderId = await redis.async.get(`transfer:${payment.id}`);
```

Comment:
Maybe put this into a utility function - i.e. hide redis impl

Reply:
Feels like an extra redundant layer. What will I name it? cache? and it'll be 1-1 redis, hard to see the point.